docs/pull_request.md

```diff
@@ -0,0 +1,158 @@
+Review such PR is the same recipe than for PR from Dependabot
+
+##### Sync analytics plan
+
+This tools import any update in the analytics plan. See instruction in the PR itself to handle it.
+More info can be found in the file [analytics.md]
+
+## Reviewing PR
+
+### Who can review pull requests?
+
+As an open source project, every one can review each others PR. Of course an approval from internal developer is mandatory for a PR to be merged.
+But comment in PR from the community are always appreciated!
+
+### What to have in mind when reviewing a PR
+
+1. User experience: is the UX and UI correct? You will probably be the second person to test the new thing, the first one is the developer.
+2. Developer experience: does the code look nice and decoupled? No big functions, etc.
+3. Code maintenance. A bit similar to point 2. Tricky code must be documented for instance
+4. Fork consideration. Is configuration of forks will be easy? Some documentation may help in some cases.
+
+### Rules
+
+#### Check the form
+
+##### PR description
+
+PR description should follow the PR template, and at least provide some context about the code change.
+
+##### File change
+
+1. Code should follow the guidelines
+2. Code should be formatted correctly
+3. XML attribute must be sorted
+4. New code is added at the correct location
+5. New classes are added to the correct location
+6. Naming is correct. Naming is really important, it's considered part of the documentation
+7. Architecture is followed. For instance, the logic is in the ViewModel and not in the Fragment
+8. There is at least one file for the changelog. Exception if the PR fixes something which has not been released yet.
+9. PR includes tests. allScreensTest when applicable, and unit tests
+10. Avoid over complicating things. Keep it simple (KISS)!
+11. PR contains only the expected change. Sometimes, the diff is showing changes that are already on develop. This is not good, submitter has to fix that up.
+
+##### Check the commit
+
+Commit message must be short, one line and valuable. "WIP" is not a good commit message.
+
+Also commit history should be nice. Having commits like "Adding temporary code" then later "Removing temporary code" is not good. The branch has to be rebased and those commit have to be dropped.
+
+PR merger could decide to squash and merge if commit history is not good.
+
+Commit like "Code review fixes" is good when reviewing the PR, since new changes can be reviewed easily, but is less valuable when looking at git history. I have no good solution to avoid this.
+
+##### Check the substance
+
+1. Test the changes!
+2. Test the nominal case and the edge cases
+3. Run the sanity test for critical PR
+
+##### Make a dedicated meeting to review the PR
+
+Some times, big PR can be hard to review. Setting up a call with the PR submitter can faster the communication, rather than putting comment and questions as GitHub comment. It has the inconvenient to not make it public.
```

Comment:
```suggestion
Sometimes a big PR can be hard to review. Setting up a call with the PR submitter can speed up the communication, rather than putting comments and questions in GitHub comment. It has the inconvenience of making the discussion non-public, consider including a summary of the main points of the "offline" conversation in the PR.
```